devops/scripts/destroy-environment.js

```diff
@@ -12,22 +12,22 @@ class DestroyEnvironment {
     if (this.environmentName === "production") {
       console.log(`Prevent deletion of environment ${this.environmentName}`);
       return;
-    } else {
-      console.log(`Deleting environment ${this.environmentName}`);
     }
 
     const config = new EnvConfig(this.environmentName);
 
     const project = await this.scaleway.findProject(config.projectName());
 
-    const namespace = await this.scaleway.findOrThrow(
-      RESOURCE.ContainerNamespace,
-      {
-        project_id: project.id,
-        name: config.containerNamespace(),
-      }
-    );
-    await this.scaleway.delete(RESOURCE.ContainerNamespace, namespace.id);
+    const namespace = await this.scaleway.find(RESOURCE.ContainerNamespace, {
+      project_id: project.id,
+      name: config.containerNamespace(),
+    });
+    if (namespace) {
```

Comment:
Inverse la condition pour éviter le else et fait un retour non ?
```suggestion
    if (!namespace) {
```

Reply:
Question de gout. Je préfere les conditions positives (ca fait moins mal au cerveau), et cela rend le branchement plus explicite